docs/modules/ROOT/pages/introduction.adoc

```diff
@@ -67,12 +67,11 @@ Additionally, prerelease version numbers may have additional suffixes, for examp
 == Requirements
 
 1. https://neo4j.com/[Neo4j Database] with https://neo4j.com/developer/neo4j-apoc/[APOC] plugin:
```

Comment:
I think we can remove the subpoints and just trim this down to the following now!

```suggestion
1. https://neo4j.com/[Neo4j Database] 4.3.2+ with https://neo4j.com/developer/neo4j-apoc/[APOC] plugin:
```